.gitignore

```diff
@@ -388,3 +388,6 @@ MigrationBackup/
 
 # Output from the "download_spec_sources.sh" script
 downloaded_sources/
+
+# build_container from the containerized-build tool
+toolkit/scripts/containerized-build/build_container/
```

Comment:
nit: `/toolkit/scripts/containerized-build/build_container/`

If you are ignoring a specific directory, then you should prefix it with `/`.